docs/minimal-metta.md

```diff
@@ -171,6 +171,35 @@ separate alternative.
 can collect the list of alternatives using `collapse-bind` filter them and
 return filtered items to the plan using `superpose-bind`.
 
+## Scope of a variable
+
+Variable name should be unique inside a separate expression evaluated. The
```

Comment:
Apologies in advance if I am overstepping.   I made a few edits for English flow, but if I changed your voice or made it less clear then feel free to reject.

```suggestion
Each separately evaluated expression is a variable scope, and therefore variable names are treated as unique inside an expression.
```